freqtrade/freqtradebot.py

```diff
@@ -1325,6 +1299,41 @@ def handle_trailing_stoploss_on_exchange(self, trade: Trade, order: Dict) -> Non
                     logger.warning(f"Could not create trailing stoploss order "
                                    f"for pair {trade.pair}.")
 
+    def manage_trade_stoploss_orders(self, trade, stoploss_orders):
```

Comment:
This will need a typehint - but more importantly - a decision on the type of stoploss_orders.

the method assumes it's a `Order` object ... but the calling method passes in a list of exchange order responses (which don't have a `.status`).

Reply:
Yes, I see, what type you think we should manage here? exchange or freqtrade orders?

Reply:
i'm ... not entirely sure about the whole `manage_trade_stoploss_orders` (as in - why it was extracted, why it needs different logic outside of the original loop (though i can imagine this somewhat).

it's probably going to be the exchange order dict - as the actual order obj is currently not available - and we really don't use it for much here.